gnat2goto/driver/goto_utils.adb

```diff
@@ -121,6 +121,23 @@ package body GOTO_Utils is
       return Ret;
    end Symbol_Expr;
 
+   procedure New_Object_Symbol_Entry (Object_Name : Symbol_Id;
```

Comment:
What's "object" in this context? We don't usually use that terminology in CBMC. I believe that's what GNAT calls an LValue? If so, this should have IsLValue set to true.

Reply:
The `object` from `do_object_declaration`.